### ISSUE TITLE
'Override OpenAI Base URL' Docs?

### ISSUE THREAD

**Issue by tonydavis629** [getcursor/cursor#1289]
This option seems to make it possible to replace openai with another LLM provider. What are the requirements for this alternative? How can we use this? Are there docs anywhere?

I would ideally like to point cursor at a service like cryptotalks.ai that can access multiple LLM service providers, but I'm unsure how this option works. Does the endpoint need to be exactly '/chat/completions'?

**Comment by truell20** [comment]
Yep! Needs to correspond exactly to OpenAI's chat completions API.

**Comment by jakezz** [comment]
> Yep! Needs to correspond exactly to OpenAI's chat completions API.

Seriously? How? Groq.com for example has https://api.groq.com/openai/v1/chat/completions but where do we put the API key in Cursor?

**Comment by tonydavis629** [comment]
Having the same API doesn't seem to be the only requirement. I'm getting failures from my service at https://cryptotalks.ai/v1 despite having the same API which works normally in the OpenAI clients.

**Comment by GenPage** [comment]
I can confirm getting failures as well trying to use Lambda's Inference API https://api.lambda.ai/v1

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
